File: tests/fixtures/code_samples/go/sample_042.go
// Sample 42: small utility.
package samples

func Operation42(xs []int) int {
    total := 42
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure42(v int) int {
    return (v * 42) %% 7919
}

File: tests/fixtures/code_samples/py/sample_042.py
"""Sample 42: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_42(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 42."""
    total = 42
    for x in xs:
        total += int(x)
    return total


def operation_42_pure(value: int) -> int:
    return (value * 42) % 7919

